### ISSUE TITLE
Move `world/series/guardian-bertha-documentaries` to new documentaries format

### ISSUE THREAD

**Issue by AshCorr** [guardian/dotcom-rendering#7989]
The [world/series/guardian-bertha-documentaries](https://www.theguardian.com/world/series/guardian-bertha-documentaries) front relies heavily on JSON.HTML thrashers for its design.

The thrashers used on the front can't really be supported in any way by DCR due to how they try to change a lot of the visual elements on the page outside of the thrashers.

Theres a few options to support this front

- Archive it
- Remove the thrashers
- Convert it to the new documentaries format

When we suggested archiving it there was a lot of pushback from DigiEd/CP, alternatively theres a new page format for the /documentaries front which doesn't rely on thrashers, maybe we can try to use that for this front?

**Comment by AshCorr** [comment]
@AshCorr talking to Lindsay Poulton on moving the front to the new format.

**Comment by rhiannareechaye** [comment]
@AshCorr I've moved this into the backlog - do you have an update on this? I've asked @HarryFischer if we could think about changing this so that it uses the tools rather than thrashers (especially now we have the video container which is styled in a nicer way?) The related ticket is: [Move /documentaries to new documentaries format#8040](https://github.com/guardian/dotcom-rendering/issues/8040)

**Comment by AshCorr** [comment]
> I've moved this into the backlog - do you have an update on this? I've asked @HarryFischer if we could think about changing this so that it uses the tools rather than thrashers (especially now we have the video container which is styled in a nicer way?) The related ticket is

We've pitched an idea for an updated front to Lindsay, its currently available on https://preview.gutools.co.uk/dotcom/test

<img width="1370" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/21217225/8f329e66-1d0a-4f45-8ed0-86401d1ece7f">

She was content with the design but there were a few things that we needed to fix:

- Make sure all the documentaries are there, "Buyers club" is currently missing.
- Make sure the "Supported by" logo is showing, which is pending on #8089 
<img width="244" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/21217225/ae9f5526-22c1-4dba-a227-64daaf1bc6dc">

**Comment by VDuczekW** [comment]
Harry is happy to revert to normal front if she’s happy (Lindsey) - removing Harry from ticket and will double check where we are with Lindsay

**Comment by VDuczekW** [comment]
blocked behind https://github.com/guardian/dotcom-rendering/issues/8089

**Comment by shesah** [comment]
@jamesgorrie looking into this. blocked by another ticket (to attach)

**Comment by rhiannareechaye** [comment]
This ticket is currently blocked by [#8089](https://github.com/guardian/dotcom-rendering/issues/8089)

**Comment by ioannakok** [comment]
Logic for "Supported by" badge has been fixed after 
* #8764 
* #8801
* #8818 

In order to show the badge in https://preview.gutools.co.uk/dotcom/test, we need to update the config of the front. Currently 
[DCR](https://preview.gutools.co.uk/dotcom/test?dcr=true) and [frontend](https://preview.gutools.co.uk/dotcom/test?dcr=false) do not show the "sponsored by" badge as expected.

`editionBranding` needs to be added on the `frontProperties`. Currently it is empty.  
![image](https://github.com/guardian/dotcom-rendering/assets/19683595/6816a9f6-01be-4037-a3fb-c262e0900f12)

For reference, see JSON of current https://www.theguardian.com/world/series/guardian-bertha-documentaries
![image](https://github.com/guardian/dotcom-rendering/assets/19683595/801078a5-f582-48cf-ad9e-ea42832ad1cc)

Not sure how and who does this technically as `world/series/guardian-bertha-documentaries` is a [`commercial`](https://fronts.gutools.co.uk/commercial/config) front. Will discuss with @HarryFischer when he's back from holidays.

**Comment by ioannakok** [comment]
Closing this ticket as the page has been reverted to a tag page: https://www.theguardian.com/world/series/guardian-bertha-documentaries

![Image](https://github.com/guardian/dotcom-rendering/assets/19683595/b2d395cd-a7b3-45e4-ab5d-3fabab13acc8)
